### PR TITLE
Login flow update and UI updated

### PR DESCRIPTION
**Pull request by PavelLaptev**
## ☕️ Reasoning

- Updated UI components structuring. In order to simplify paths to components removed component groups like `inputs\...`. Path to UI components should be simple unless it's a complex ciomponent like `SegmentController` which consist of two components but you can't use them separatly.  Other components should be in the root folder.
- Added an option to copy the login link.

image

## 🧢 Changes

- Many small changes for paths
- New component `LinkButton`
- Updated `login` functions. Added new function `loginAndCopyLink`

## 🎫 Affected issues

Issue https://github.com/gitbutlerapp/gitbutler/issues/2650

### CODE REVIEW

File: apps/desktop/src/lib/utils/clipboard.ts [hunk 1]
```diff
@@ -4,13 +4,15 @@ export function copyToClipboard(text: string) {
 	if (!navigator.clipboard) {
 		toasts.error('Clipboard API not available');
 	} else {
+		console.log('Copying to clipboard:', navigator.clipboard);
```
Comment (ndom91) [inline apps/desktop/src/lib/utils/clipboard.ts]: Left over?

**Comment by ndom91** [comment]
Seems like there are some import-order eslint issues and both `/commitLines` and `/CommitLines` are both still present. Same deal with `segmentControl`/`SegmentControl`

File: apps/desktop/src/lib/utils/clipboard.ts (hunk 1 above)
Comment (PavelLaptev) [reply apps/desktop/src/lib/utils/clipboard.ts]: yes, thanks for noticing :-)

File: packages/ui/src/stories/modal/DemoModal.svelte [hunk 2]
```diff
@@ -14,11 +14,16 @@
 	let modal = $state<Modal>();
 </script>
 
-<Button on:click={() => modal?.show()}>Show</Button>
+<Button
+	onclick={() => {
+		console.log('show');
```
Comment (mtsgrd) [inline packages/ui/src/stories/modal/DemoModal.svelte]: Just scanning this PR, noticed this should probably be removed.

Comment (PavelLaptev) [reply packages/ui/src/stories/modal/DemoModal.svelte]: Done :-)

**Comment by PavelLaptev** [comment]
> Seems like there are some import-order eslint issues and both `/commitLines` and `/CommitLines` are both still present. Same deal with `segmentControl`/`SegmentControl`. But otherwise looks alright 👍

There was a casing issue. Fixed that

**Review by mtsgrd — APPROVED** [review]

**Pull request merged**